.gitignore

```
@@ -39,6 +39,6 @@ Testing/*
 cmake-build-debug
 cmake-build-release
 build
-external/vcpkg
+external/*
```

This way, `external/CMakeLists.txt` would be ignored. Please try to add an exception to the rule (and test it):
```suggestion
external/*
!external/CMakeLists.txt
```

---

Great catch! Tested your suggestion and it works.